Day-I-Social-Card/social-card/src/components/HeaderComponents/ImageThumbnail.js

```diff
@@ -1,2 +1,16 @@
 import React from 'react';
 import './Header.css';
+
+const img = () => (
+
+  <img
+    className='img'
+    src='https://pbs.twimg.com/profile_images/973277209644249089/0Te2jtBH_400x400.jpg'
+    alt='Lambda schol logo'
+  />
+
+);
+
+
+
+export default img;
```

Comment:
Usually, for best practices, you'll want to keep your naming conventions across files. You're exporting it here as img but importing it as ImageThumbnail.